controllers/movies.js

```diff
@@ -98,7 +98,7 @@ module.exports.postComment = async (req, res) => {
 
   try {
     const comment = await commentService.postComment(id, content, author);
-    res.status(201).json({ response: comment });
+    res.status(201).json({ response: [ comment ] });
```

Comment:
I was honing for consistency in the response but since like you said it's just one object being returned then no need to array. So i have removed the quare brackets